ios/RNPlayerViewManager.swift

```diff
@@ -105,6 +105,28 @@ class RNPlayerViewManager: RCTViewManager {
         }
     }
 
+    @objc func setPictureInPicture(_ viewId: NSNumber, enterPictureInPicture: Bool) {
+        bridge.uiManager.addUIBlock { [weak self] _, views in
+            guard
+                let self,
+                let view = views?[viewId] as? RNPlayerView
+            else {
+                return
+            }
+            guard let playerView = view.playerView else {
+                return
+            }
```

Comment:
Is this not possible?

```suggestion
            guard
                let self,
                let view = views?[viewId] as? RNPlayerView,
                let playerView = view.playerView
            else {
                return
            }
```

Reply:
fixed in https://github.com/bitmovin/bitmovin-player-react-native/pull/289/commits/5e20a616a1b0b225b5268e21e1188017625593bc